blocktx/client.go

```diff
@@ -57,9 +74,9 @@ func (btc *Client) Start(minedBlockChan chan *blocktx_api.Block) {
 	for {
 		stream, err := btc.client.GetBlockNotificationStream(context.Background(), &blocktx_api.Height{})
 		if err != nil {
-			btc.logger.Errorf("Error getting block notification stream: %v", err)
+			btc.logger.Error("Error getting block notification stream", slog.String("err", err.Error()))
```

Comment:
I think Errorf would fit better here. 
Also you don't have to explicitly call .Error() on error when passed to formatter.

Reply:
`slog` does not provide `Errorf()` function

Reply:
I see. no problem then.